controllers/util/util.go

```diff
@@ -198,3 +198,13 @@ func GetRouteIfAvailable(ctx context.Context, routeName, ns string, client clien
 	}
 	return true, route, nil
 }
+
+func GetMlmdGrpcCertificatesSecret(ctx context.Context, client client.Client, namespace string, dspaName string) (v1.Secret, error) {
```

Comment:
I feel we should make this generic, ie simply `GetSecret`, and take in the name of the secret as a parameter instead of hard-coding it here (especially since this in in `util.go`, which contains mostly generic helper functions)

Reply:
+1. I believe that's what @HumairAK meant when he previously said

> this can be a static helper method [like we do for configmaps](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/6f28be9771577deb0cc0cc7c12a4ca690635f900/controllers/util/util.go#L87-L99)

Reply:
Done.